src/views/GroupView.vue

```diff
@@ -1,73 +1,669 @@
 <template>
   <b-container fluid class="admin">
-    <b-jumbotron class="jumbotron" fluid>
+    <b-jumbotron class="group-jumbotron" fluid>
       <div>
-        <b-link
-          class="back-link"
-          :to="{
-            name: 'groups',
-          }"
-        >
+        <b-link class="back-link" :to="{ name: 'groups' }">
           <font-awesome-icon icon="angle-left" size="xs" />
           <span>Back to groups</span>
         </b-link>
       </div>
       <h1>
         <font-awesome-icon icon="users" size="xs" />
-        <span v-if="!fetched" class="name-placeholder">loading group name</span>
-        <span v-else>
-          {{ group && group.groupname }}
-        </span>
+        <span>{{ groupName }}</span>
       </h1>
       <p class="lead">
         Manage the users associated with this group and view the devices
         associated with it.
       </p>
     </b-jumbotron>
+    <b-tabs
+      card
+      class="group-tabs"
+      nav-class="container"
+      v-model="currentTabIndex"
```

Comment:
I think it would be nice if each tab was in a different component.   As far as I can tell they aren't related